workflow/executor/common/wait/wait.go

```diff
@@ -26,22 +26,32 @@ func UntilTerminated(kubernetesInterface kubernetes.Interface, namespace, podNam
 }
 
 func untilTerminatedAux(podInterface v1.PodInterface, containerID string, listOptions metav1.ListOptions) (bool, error) {
+	for {
+		complete, done, err := doWatch(podInterface, containerID, listOptions)
```

Comment:
In English, "complete" means the same thing as "done". Is there a better name to use?

Reply:
Very fair point. I couldn't think of a better synonym, so I went for something more explicit and swapped the booleans around to match.